EDEngineer/Resources/Data/blueprints.json

```diff
@@ -16406,5 +16406,65 @@
       }
     ],
     "Grade": 5
+  },
+  {
+    "Type": "The Sarge",
+    "Name": "unlock",

```

Comment:
could you swap these two? Type = Unlock and Name = The Sarge
it makes more sense that way